workers/unity/Packages/io.improbable.gdk.core/Systems/EntitySystem.cs

```diff
@@ -49,6 +51,11 @@ internal void ApplyDiff(ViewDiff diff)
                 {
                     entitiesRemoved.Add(entityId);
                 }
+
+                if (entitiesAdded.Count != 0 || entitiesRemoved.Count != 0)
```

Comment:
This still feel very hacky. Unity does have a concept of a `Version` on every system, which changes whenever they run.

Reply:
Discussed offline, this is okay for now, will add a note and raise a ticket to revisit this (and the `EntitySystem` more broadly) in the future.

Reply:
Raised https://improbableio.atlassian.net/browse/UTY-2606